### DISCUSSION TITLE
MIRYOKU_LAYOUT=vi Does Not Work

### DISCUSSION THREAD

**Discussion by raviolispy** [manna-harbour/miryoku#193] [Q&A] [answered]
Using the below command, I can successfully compile for my iris rev7 (although curiously, I had to remove some things from the default version so the hex file was small enough). The other options I specified such as the QWERTY layout all work but the MIRYOKU_NAV=vi doesn't appear to shift the navigation keys to hjkl. Can anyone help please?

`qmk compile -c -kb keebio/iris/rev7 -km manna-harbour_miryoku  -e MIRYOKU_ALPHAS=QWERTY  -e MIRYOKU_CLIPBOARD=WIN  -e MIRYOKU_TAP=QWERTY -e MIRYOKU_NAV=vi`

**Comment by manna-harbour** [comment, marked as answer]
Unfortunately due to technical limitations the options are case sensitive and no warning is given if an unrecognised option is given. The options are all uppercase, so please use `MIRYOKU_NAV=VI` instead. For workflow builds, options are case-insensitive. For the firmware size issue see https://github.com/qmk/qmk_firmware/blob/master/docs/squeezing_avr.md.
